### DISCUSSION TITLE
react native reanimated 2.10.0 downloaded downloadBoost 124.24 MB

### DISCUSSION THREAD

**Discussion by putuoka** [software-mansion/react-native-reanimated#3617] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/20880839/192684516-b86fe796-511f-423d-aaed-ed373acc21e2.png)

very long wait download

**Comment by tomekzaw** [comment, marked as answer]
Duplicate of https://github.com/software-mansion/react-native-reanimated/issues/3523, will be fixed in https://github.com/software-mansion/react-native-reanimated/pull/3602.
